### PR TITLE
feature: Update registries with new region account number mappings.

### PR DESCRIPTION
**Pull request by kenny-ezirim**
*Issue #, if available:*
SageMaker support for region expansion

*Description of changes:*
Updating json files with mappings of new regions and associated account numbers.

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/mxnet.json [hunk 1]
```diff
@@ -408,14 +408,17 @@
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
                     "ap-south-1": "520713654638",
+                    "ap-south-2": "772153158452",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:411]: Please remove these entries for all framework versions whose repository is `sagemaker-mxnet`, `sagemaker-mxnet-serving`, `sagemaker-mxnet-eia`, and `sagemaker-mxnet-serving-eia`.

File: src/sagemaker/image_uri_config/pytorch.json [hunk 2]
```diff
@@ -217,13 +220,15 @@
...
                     "ap-southeast-2": "763104351884",
                     "ap-southeast-3": "907027046896",
                     "ca-central-1": "763104351884",
                     "cn-north-1": "727897471807",
                     "cn-northwest-1": "727897471807",
                     "eu-central-1": "763104351884",
+                    "eu-central-2": "380420809688",
                     "eu-north-1": "763104351884",
                     "eu-west-1": "763104351884",
                     "eu-west-2": "763104351884",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/pytorch.json:234]: eu-south-2 is missing here.

File: src/sagemaker/image_uri_config/tensorflow.json [hunk 3]
```diff
@@ -298,14 +298,17 @@
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
                     "ap-south-1": "520713654638",
+                    "ap-south-2": "772153158452",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/tensorflow.json:301]: Please remove these new entries from the framework versions whose repository is `sagemaker-tensorflow` and `sagemaker-tensorflow-serving`.

Please also add these regions to the framework versions that have the repo name `tensorflow-inference-eia`.

File: src/sagemaker/image_uri_config/tensorflow.json [hunk 4]
```diff
@@ -637,6 +670,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/tensorflow.json:673]: Please remove this.

Comment (kenny-ezirim) [reply src/sagemaker/image_uri_config/tensorflow.json]: Done.

**Review by saimidu — APPROVED** [review]

**Review by knikure — APPROVED** [review]

**Pull request merged**